jshell.support/src/org/netbeans/modules/jshell/project/ShellProjectUtils.java

```diff
@@ -372,11 +350,12 @@ public static ClassPath projecRuntimeClassPath(Project project) {
                 ShellProjectUtils.findProjectImportedModules(project, 
                     ShellProjectUtils.findProjectModules(project, null))
         );
+        
```

Comment:
The return value of launchVMOptions is actually (sometimes) checked for null; I agree with changing it to @NonNull, but in that case, org.netbeans.modules.jshell.env.JShellEnvironment should be modified not to call JShell.Builder.removeVMOptions. The null check can be removed in the other callees.